bigbluebutton-html5/imports/ui/components/poll/component.jsx

```diff
@@ -267,29 +322,151 @@ class Poll extends Component {
   }
 
   renderPollOptions() {
-    const { isMeteorConnected, intl } = this.props;
```

Comment:
(continued from the first comment) -> The line 318 could be modified as: 
handleBackClick={this.handleBackClick.bind(this)}
(adding ".bind(this)")